app-modules/service-management/tests/Filament/ServiceMonitoring/Pages/CreateServiceMonitoringTest.php

```diff
@@ -114,10 +114,6 @@
             ServiceMonitoringTargetRequestFactory::new()->state(['domain' => str()->random(256)]),
             ['domain' => 'max'],
         ],
-        'domain url' => [
-            ServiceMonitoringTargetRequestFactory::new()->state(['domain' => 'invalid-url']),
-            ['domain' => 'url'],
-        ],
```

Comment:
I would maybe just add a comment here mentioning that `domain url` validation is tested more extensively in a test below.